test/modules/linear_boltzmann_solvers/transport_steady/tests.json

```diff
@@ -515,5 +515,43 @@
         "abs_tol": 1.0e-6
       }
     ]
+  },
+  {
+    "file": "1g_infinite_pure_absorber_balance_gmres.lua",
+    "comment": "Infinite, 1g, pure absoreber, with balance using GMRES",
+    "num_procs": 3,
+    "checks": [
+      {
+        "type": "KeyValuePair",
+        "key": "[0]   In-flow rate                =",
+        "goldvalue": 1.553633e+02,
+        "abs_tol": 1.0e-6
+      },
+      {
+        "type": "KeyValuePair",
+        "key": "[0]   Out-flow rate               =",
+        "goldvalue": 1.553633e+02,
+        "abs_tol": 1.0e-6
+      }
+    ]
+  },
+  {
+    "file": "1g_infinite_pure_absorber_balance_richardson.lua",
+    "comment": "Infinite, 1g, pure absoreber, with balance using Richardson",
+    "num_procs": 3,
+    "checks": [
+      {
+        "type": "KeyValuePair",
+        "key": "[0]   In-flow rate                =",
+        "goldvalue": 1.965343e+02,
+        "abs_tol": 1.0e-6
+      },
+      {
+        "type": "KeyValuePair",
+        "key": "[0]   Out-flow rate               =",
+        "goldvalue": 1.965343e+02,
```

Comment:
I do not get it: why is the golden value for richardson 1.96e2 while it is 1.55e2 for GMRes. Are you not solving the same problem??

Reply:
It is the same problem. I chalked this up to differences in the iterative process. No?

Reply:
@wdhawkins Not possible! The solution is uniform and isotropic, Phi = Q/sigma_a, and Psi = Phi/4pi. So, the leakage are the half-range integrals of a constant, isotropic value on each face, so both methods need to give the same leakage! 196 versus 155 is a HUGE difference.

Reply:
And, I've noticed that richardson gives a different answer depending on the number of processors.

Reply:
@wdarylhawkins : the exact answer is, for your problem: 150 / true_pi * half_range_integral
where the half range integral is Omega dot n integrated over the half range. This value gets closer and closer to true_pi as you add more directions. So, I would trust more the value from GMRES, which is 155,... with 32 directions.

Two things:
1. you need to fix Richardson
2. please verify that, as the # of directions increases, the lekage tends towards 150.00000

Reply:
#2 certainly looks to be true with GMRES. At 32k angles, leakage is 1.500403e+02. At 64k angles, it's 1.500102e+02.

Reply:
Not sure what's up with Richardson. Even without reflecting bcs, something seems off. I'll start looking at it.

Reply:
Small update. This bug is not something that was introduced in OpenSn. It looks like it's a chi-tech bug.

Reply:
I believe I have it fixed. Let me do a bit of testing, but we should discuss.

Reply:
#306 fixes the issue with Richardson. I've also verified that leakages tends towards 150 as we refine the number of angles for the Richardson test.